rust/src/blocks/cb_csv.rs

```diff
@@ -0,0 +1,266 @@
+use crate::block::Block;
+use crate::core::do_blocking;
+use crate::core::log;
+use crate::core::registerBlock;
+use crate::types::common_type;
+use crate::types::ClonedVar;
+use crate::types::Context;
+use crate::types::ParamVar;
+use crate::types::Parameters;
+use crate::types::Seq;
+use crate::types::Table;
+use crate::types::Type;
+use crate::types::SEQ_OF_STRINGS_TYPES;
+use crate::types::STRING_TYPES;
+use crate::CString;
+use crate::Types;
+use crate::Var;
+use core::convert::TryFrom;
+use core::convert::TryInto;
+use csv::Reader;
+use csv::ReaderBuilder;
+use csv::WriterBuilder;
+
+lazy_static! {
+  static ref PARAMETERS: Parameters = vec![
+    (
+      cstr!("NoHeader"),
+      cbccstr!("If the block should not skip parsing the first row."),
```

Comment:
Double-negations are hard to read.

"Whether the block should parse the first row as data, instead of header".

Reply:
Good suggestion